src/CommandingOfficers/Tech.java

```diff
@@ -267,7 +260,7 @@ private TeleportEvent generateDropEvent(MapMaster gameMap, Set<XYCoord> priorDro
       {
         // Record any locations near owned properties; we want to be able to rescue unprotected structures.
         for( XYCoord xyc : Utils.findLocationsInRange(gameMap, propCoord, 0, dropRange) )
-          if( !unitModelToDrop.propulsion.canTraverse(gameMap.getEnvironment(xyc)) )
+          if( unitModelToDrop.propulsion.canTraverse(gameMap.getEnvironment(xyc)) )
```

Comment:
Hah. I thought that one was odd, but I can't say I understand exactly what all the bits are doing so I didn't wanna say anything lol